koku/masu/processor/report_parquet_processor_base.py

```diff
@@ -147,14 +148,14 @@ def _generate_create_table_sql(self, partition_map=None):
                 f") WITH(external_location = 's3a://{s3_path}', format = 'PARQUET',"
                 " partitioned_by=ARRAY['source', 'year', 'month'])"
             )
-        LOG.info(f"Create Parquet Table SQL: {sql}")
+        LOG.info(log_json(msg="attempting to create parquet table", table=self._table_name, schema=self._schema_name))
```

Comment:
We should move this into the `create_table` func right above the `_execute_sql` call. This message doesn't really belong in this func.

Reply:
Sure